lib/corundum/default_configuration/skel-files/rakefile

```diff
@@ -15,24 +15,19 @@ module Corundum
         content.type = type
       end
     end
+
     rspec = RSpec.new(core)
-    cov = SimpleCov.new(core, rspec) do |cov|
-      cov.threshold = 70
+    SimpleCov.new(core, rspec) do |cov|
+      cov.threshold = 85
     end
 
     gem = GemBuilding.new(core)
-    cutter = GemCutter.new(core,gem)
-    email = Email.new(core)
-    vc = Git.new(core) do |vc|
+    GemCutter.new(core,gem)
```

Comment:
I'd suggest this template is a good candidate for extensive in-line documentation explaining what each line does, what its defaults are, and what options are available. Especially given that instantiating classes (as opposed to declaring tasks and namespaces) is a generally unfamiliar syntax for a Rakefile.  This is typically the file that most throws me when trying to work on a Corundum gem.

Consider the templated config/routes.rb in Rails and the templated spec_helper.rb by rSpec rails as examples of what I mean.